python/tests/unit/arcticdb/version_store/test_normalization.py

```diff
@@ -477,3 +477,16 @@ def test_no_inplace_index_array_modification(lmdb_version_store, sym, datetime64
     lmdb_version_store.write(sym, pandas_container)
     assert pandas_container.index is original_index_array
     assert pandas_container.index.dtype == datetime64_dtype
+
+
+def test_pyarrow_error(lmdb_version_store):
+    _ = pytest.importorskip("pyarrow")
+
+    error_msg_intro = "PyArrow-backed pandas DataFrame and Series are not currently supported by ArcticDB."
+    df = pd.DataFrame(data=[[1.0, 0.2], [0.2, 0.5]], dtype="float32[pyarrow]")
```

Comment:
Presumably it's not possible to have a mixed Arrow/numpy backed df? Would be interesting to test that, if it's possible

Reply:
> Presumably it's not possible to have a mixed Arrow/numpy backed df?

Good suggestion. This is tested in d350679.

Reply:
> Presumably it's not possible to have a mixed Arrow/numpy backed df?

Good suggestion. This is tested in 3c986a6edb8e6f91412aeef317aea31fffb8936e.